xiaogpt/xiaogpt.py

```diff
@@ -175,6 +175,8 @@ def chatbot(self):
                 self._chatbot = ChatGPTBot(
                     self.config.openai_key, self.config.api_base, self.config.proxy
                 )
+            elif self.config.bot == "edgegpt":
+                self._chatbot = EdgeGPTBot(self.config.proxy, **self.config.gpt_options)
```

Comment:
```suggestion
                self._chatbot = EdgeGPTBot(self.config.proxy, self.config.bing_cookies)
```

bing_cookies是必填参数，应该有一个单独的配置项（抱歉如果之的评论有误）